web/assets/js/widgets/core/snowplow-tracking.js

```diff
@@ -83,6 +83,10 @@
       }
     };
 
+    const stringToPascalCase = (value) => {
```

Comment:
```suggestion
    const stringToKebabCase = (value) => {
```

I think it is Kebab case 😉 

https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
You're right 🐑